apps/framework-cli/src/cli/settings.rs

```diff
@@ -243,3 +248,34 @@ machine_id="{{uuid}}"
     }
     Ok(())
 }
+
+impl Settings {
+    /// Checks if container shutdown should be skipped based on settings and environment variables
+    ///
+    /// This function intelligently interprets the MOOSE_SKIP_CONTAINER_SHUTDOWN environment variable:
+    /// - Values like "1", "true", "yes" (case-insensitive) are interpreted as true
+    /// - Values like "0", "false", "no" (case-insensitive) are interpreted as false
+    /// - If the environment variable is not set, uses the value from config
+    pub fn should_skip_container_shutdown(&self) -> bool {
+        // Check environment variable first (takes precedence over config)
+        match std::env::var("MOOSE_SKIP_CONTAINER_SHUTDOWN") {
```

Comment:
Not sure this is needed - in the same file we load the settings from the env variables directly. (line 171-181)

Reply:
I think this is worth cleaning up. Updating PR now.

Reply:
@callicles the PR is update with this cleanup.  It's ready for a final review.

Reply:
Should this implementation be removed then? it looks like it is not used anymore?